stable/cloudserver/templates/api-configmap.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "cloudserver.fullname" . }}
+  labels:
+    app: {{ template "cloudserver.name" . }}
+    chart: {{ template "cloudserver.chart" . }}
+    component: api
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+data:
+{{- if .Values.api.locationConstraints }}
```

Comment:
maybe you can create this configmap if the `{{.Values.api.locationConstraints }}` exists, if you not define that variable it will create an empty configmap.

Reply:
I decided to make locationConstraints mandatory since, without it, the application is pretty much useless.